app/views/Project/EntryEdit/index.tsx

```diff
@@ -615,7 +620,7 @@ function EntryEdit(props: Props) {
                             variables: {
                                 projectId,
                                 deleteIds: entryDeleteIds,
-                                entries: transformedEntries,
+                                entries: transformedEntries.reverse(),
```

Comment:
We don't need to reverse anymore. If we do we can use the `reverse` prop on ListView, without modifying the data.